src/main/treeDataProviders/utils/dependencyUtils.ts

```diff
@@ -203,25 +215,16 @@ export class DependencyUtils {
             scanProgress.onProgress
         )
             .then((issuesFound: number) => {
```

Comment:
Using `await` on 209 with `then` on line 217 seems wrong to me. Let's refactor this and try to avoid the duplication of `issuesCount` and `issuesFound`